agent/consul/fsm_data_store.go

```diff
@@ -0,0 +1,77 @@
+package consul
```

Comment:
You'll probably want to rebase here? I don't think you meant to touch these files.